azure-kusto-data/azure/kusto/data/kcsb.py

```diff
@@ -11,6 +11,8 @@
         https://github.com/Azure/azure-kusto-python/blob/master/azure-kusto-data/tests/sample.py
     """
 
+    kcsb_invalid_item_error = "%s is not supported as an item in KustoConnectionStringBuilder"
```

Comment:
Make this a proper exception that lives in exceptions.py (inherit from keyerror to not break interface)

Reply:
we'll do a separate PR for kcsb exceptions